server/routes/v1/user.ts

```diff
@@ -1,60 +1,140 @@
 import express, { Request, Response } from "express";
 import moment from "moment";
-import EmailService from "../../services/emailService";
+import jwt from "jsonwebtoken";
 
-import db from "../../models/index";
+import EmailService from "../../services/emailService";
+import db from "../../models";
 import UserController from "../../controllers/v1/user";
+import { User } from "../../models/user";
 
-const router = express.Router();
+const userRouter = express.Router();
 
-export const userController: UserController = new UserController(db.User, new EmailService());
-let response;
+const uContr: UserController = new UserController(
+  db.User,
+  new EmailService()
+);
 
-export async function signInHandler(
-  req: Request,
-  res: Response
-) {
-  const { userName, password } = req.body;
-  response = await userController.signIn(userName, password);
+let response;
+let user: User | undefined;
 
-  if (response.data.token) {
-    res.cookie("token", JSON.stringify(response.data.token), {
+async function signOutHandler(req: Request, res: Response) {
+  res
+    .clearCookie("token", {
       httpOnly: true,
-      expires: moment().add(1, "hour").toDate(),
       signed: true,
-    });
-  }
+    })
+    .status(200)
+    .json({ message: "Signed out successfully" });
+}
 
-  res.status(response.status).json(response.data);
+export async function signInHandler(req: Request, res: Response) {
+  try {
+    const { userName, password } = req.body;
+
+    if (!userName || !password) {
+      res.status(400).json({ message: "Username or password is undefined" });
```

Comment:
You could use a ternary operator here and specify whether it is username, password, or both. Would make the message more helpful